_visual/typography.md

```diff
@@ -175,8 +176,8 @@ Something about load performance consideration and how we recommend one of the f
             <p class="usa-monospace">
               font-family: ‘Merriweather’<br>
               font-weight: 400<br>
-              font-size: 22px<br>
-              line-height: 1.375em/30px
+              font-size: 20px<br>
+              line-height: 1.7em/34px
```

Comment:
@angel oh cool, are the style changes something we want to add in to this PR or a different one?